src/index.html

```diff
@@ -1,17 +1,118 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post"
+      onsubmit="onSubmit()">
+      <fieldset class="block personal-information">
+        <legend>Personal information</legend>
+        <div class="form-field">
+          <label for="surname">Surname:</label>
+          <input type="text"
+          name="surname"
+          id="surname" autocomplete="off">
+        </div>
+        <div class="form-field">
+          <label for="name">Name:</label>
+          <input type="text" name="name"
+          id="name" autocomplete="off">
+        </div>
+        <div class="form-field">
+          <label for="how-old">How old are You?</label>
+          <input type="number" name="age"
+          id="how-old" value="12"
+          min="1" max="100">
+        </div>
+        <div class="form-field">
+          <label for="full-date">Full date of birth:</label>
+          <input type="date" name="date-of-birth"
+          id="full-date">
+        </div>
+        <div class="form-field last-item">
+          <label for="term">I accept the term of the agreement</label>
+          <input type="checkbox" name="accept"
+          id="term">
+        </div>
+      </fieldset>
+      <fieldset class="block registartion">
+        <legend>Registration</legend>
+        <div class="form-field">
+          <label for="email">E-mail:</label>
+          <input type="email" name="email-link"
+          id="email" placeholder="email@example.com"
+          required>
+        </div>
+        <div class="form-field last-item">
+          <label for="password">Password:</label>
+          <input type="password" name="password"
+          id="password" minlength="8"
+          maxlength="12">
```

Comment:
```suggestion
          <input
            type="password"
            name="password"
            id="password"
            minlength="8"
            maxlength="12"
          >
```
Форматируй теги таким образом

Reply:
Відфоматував, але в мене така проблема, коли включаю автоформатуваня вс код shift +alt +F , то всі атрибути повертаються в один рядок, якось можна це настроїти в автоформатуванні, щоб воно автоматично не переводило всі атрибути назад в один рядок?

Reply:
Возможно у тебя стоит экстеншн prettier, который это делает